marimbabot_vision/scripts/flatten_whiteboard_data.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Maybe sort the imports. There is an option for it in vs code if you right-click the code.

Reply:
Done. Thanks for the info!